.travis-ci.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+case $TRAVIS_OS_NAME in
+  linux)
+    sudo apt-get install coinor-libclp-dev coinor-libcbc-dev coinor-libsymphony-dev mccs
+    ;;
```

Comment:
It could be, but Travis seem to have taken their linter offline while they develop a new one and I could not be bothered to figure out the expansion of the matrix so that the addons only applied to Linux, and not macOS. It's also the case that once the backend testing is extended to macOS that it has to be done using commands rather than YAML changes, because you need a different homebrew tap. So it seemed better to do all the package installations by hand.

It'd be fine if YAML weren't such a arm-gnawingly useless way of specifying anything...